component/org.wso2.carbon.identity.dpop/src/main/java/org/wso2/carbon/identity/dpop/validators/DPoPHeaderValidator.java

```diff
@@ -56,19 +56,34 @@ public class DPoPHeaderValidator {
 
     static final Log log = LogFactory.getLog(DPoPHeaderValidator.class);
 
+    public static class MultipleDPoPHeadersException extends RuntimeException {
```

Comment:
why we need nested class?
Why we need to extend Runtime exception
Please recheck the class names